src/sql/parts/notebook/notebook.component.ts

```diff
@@ -3,8 +3,6 @@
 *  Licensed under the Source EULA. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-import 'sql/parts/notebook/notebookStyles';
```

Comment:
Moving to only import once, in the notebookService. This is because this code now has to be setting aware (plus it's more efficient as we were registering theming participants multiple times)

Reply:
👍